lib/routes/home/widgets/payments_filter/payment_filter_exporter.dart

```diff
@@ -14,7 +14,7 @@ import 'package:share_plus/share_plus.dart';
 
 class PaymentmentFilterExporter extends StatelessWidget {
   final _log = Logger("PaymentmentFilterExporter");
-  final PaymentTypeFilter filter;
+  final List<PaymentTypeFilter> filter;
```

Comment:
Make optional and rename to filters
```suggestion
  final List<PaymentTypeFilter>? filters;
```

Reply:
Added